### ISSUE TITLE
Link to the accessibility help request form not found

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2878]
### Expected behavior
The catalog has a link to [the accessibility office form](http://accessibility.princeton.edu/help) in the footer.

### Actual behavior
There is no link to [the accessibility office form](http://accessibility.princeton.edu/help).

### Steps to replicate

1. Go to the page
2. Examine the links in the footer

### Impact of this bug

Users cannot easily report accessibility issues

### Implementation notes, if any

* The [Library Website](https://library.princeton.edu/) has a good example of a link to this form in its footer
* It should indicate to screen readers that it opens in a new tab.

**Comment by christinach** [comment]
The accessibility help link redirects to [The page you were looking for doesn't exist](https://catalog.princeton.edu/%7B%7B%20link.href%20%7D%7D).

**Comment by christinach** [comment]
This is a bug in Lux
